apps/prairielearn/src/components/InstructorInfoPanel.html.ts

```diff
@@ -74,26 +74,36 @@ export function InstructorInfoPanel({
       <div class="card-header bg-warning">
         <h2>Staff information</h2>
       </div>
-      <div class="card-body">
-        ${InstanceUserInfo({ instance_user, instance_group, instance_group_uid_list })}
-        ${QuestionInfo({
+      ${ListGroup([
+        InstanceUserInfo({ instance_user, instance_group, instance_group_uid_list }),
+        QuestionInfo({
           course,
           course_instance,
           question,
           variant,
           question_is_shared,
           questionContext,
-        })}
-        ${VariantInfo({ variant, timeZone, questionContext })}
-        ${IssueReportButton({ variant, csrfToken, questionContext })}
-        ${AssessmentInstanceInfo({ assessment, assessment_instance, timeZone })}
-        ${ManualGradingInfo({ instance_question, assessment, questionContext })}
-      </div>
+        }),
+        VariantInfo({ variant, timeZone, questionContext }),
```

Comment:
The variant info doesn't have a heading, I believe it was intended to be shown in conjunction with the question info. I'd suggest either add a heading or combine both (call VariantInfo from QuestionInfo?).

Reply:
I'm bike shedding so hard now, but if we do combine them then please just make it a combined `QuestionVariantInfo()` function so that one function === one list item. Adding a `Variant:` header would also be fine with me and might be clearer.

Reply:
Added in 92d62a9. I updated the other headings to use sentence case while I was at it.